fbu_twitter_starter/twitter/Models/Tweet.h

```diff
@@ -0,0 +1,33 @@
+//
+//  Tweet.h
+//  twitter
+//
+//  Created by Abigail Shilts on 6/20/22.
+//  Copyright © 2022 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "User.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Tweet : NSObject
```

Comment:
nit: In objective c we add prefixes for class names to avoid name conflicts, for example CPTweet. In swift that is not necessary, because swift supports namespaces.